README.md

```diff
@@ -12,12 +13,13 @@ This program is aimed at short startup time, with all generally used features.
 This is a evince or qpdfview alternative for linux users.
 
 ### Installation
-To Install this program open terminal inside gospel-pdf-viewer-master directory.  
+To Install this program open terminal inside gospel-pdf-viewer directory.  
```

Comment:
@dammhannes I expect users to download zip from the github and then extract it (not git clone the repo).
so the directory is gospel-pdf-viewer-master and not gospel-pdf-viewer.
Hence, you dont need to change this line.

Reply:
I didn't look at it from that perspective, because I thought if users are experienced enough to compile files, they would rather git clone a repo. But no worries, they would also figure out which folder they need to run the terminal in ;)